tick/array_test/tests/array_memory_test.py

```diff
@@ -2,12 +2,29 @@
 
 # -*- coding: utf8 -*-
 
+from tick.array.build.array import tick_double_array_to_file, tick_double_sparse2d_to_file
+from tick.array.build.array import tick_double_array_from_file, tick_double_sparse2d_from_file
+
+import sys
 import unittest
+import scipy
 import numpy as np
 from tick.array_test.build import array_test as test
 import weakref
 from scipy.sparse import csr_matrix
 
+import gc
+
+def dump_garbage():
+
+    gc.collect()
+    print("\nGARBAGE:")
+    print("\nGARBAGE OBJECTS:")
+    for x in gc.garbage:
+        s = str(x)
+        if len(s) > 80: s = s[:80]
+        print(type(x),"\n  ", s)
```

Comment:
Tests should not print :)

Reply:
That function is never called, it was for debugging GC while trying to figure out if SparseArray2d was being deleted properly. Can remove though,